impeller/entity/contents/content_context.cc

```diff
@@ -38,10 +38,19 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
 
   switch (pipeline_blend) {
     case BlendMode::kClear:
-      color0.dst_alpha_blend_factor = BlendFactor::kZero;
-      color0.dst_color_blend_factor = BlendFactor::kZero;
-      color0.src_alpha_blend_factor = BlendFactor::kZero;
-      color0.src_color_blend_factor = BlendFactor::kZero;
+      if (is_for_rrect_blur_clear) {
+        color0.alpha_blend_op = BlendOperation::kReverseSubtract;
+        color0.color_blend_op = BlendOperation::kReverseSubtract;
+        color0.dst_alpha_blend_factor = BlendFactor::kOne;
+        color0.dst_color_blend_factor = BlendFactor::kOne;
+        color0.src_alpha_blend_factor = BlendFactor::kDestinationColor;
+        color0.src_color_blend_factor = BlendFactor::kDestinationColor;
```

Comment:
I think this will cause the clear to weaken as the paint color becomes transparent. Is that behavior correct?

Reply:
That's right, thats how you get the blurred results (that work locally):
<img width="578" alt="Screenshot 2023-09-26 at 8 50 09 AM" src="https://github.com/flutter/engine/assets/30870216/0040d74e-94d4-48c7-83db-56f7da831959">

Reply:
Tried this out with wacky transparent colors in your repro and this seems to work great!